src/components/surveys/survey-design/question-edit/rhs-subcontrols/Time.tsx

```diff
@@ -57,12 +58,11 @@ const ValueSelector: React.FunctionComponent<{
 
       <StyledDropDown
         labelId={CONFIG[type].labelId}
-        value={value}
+        value={value || ''}
```

Comment:
When `value` is `undefined`, MUI warns that an 'out-of-range value' has been specified for the select component and recommends passing an empty string in this case.